test/e2e/suite/conformance/certificates/suite.go

```diff
@@ -188,6 +193,120 @@ func (s *Suite) Define() {
 			err = f.Helper().WaitCertificateIssuedValid(f.Namespace.Name, "testcert", time.Minute*5)
 			Expect(err).NotTo(HaveOccurred())
 		})
+
+		It("should issue a certificate that defines a commonName and sets a duration", func() {
+			s.checkFeatures(DurationFeature)
+
+			testCertificate := &cmapi.Certificate{
+				ObjectMeta: metav1.ObjectMeta{
+					Name:      "testcert",
+					Namespace: f.Namespace.Name,
+				},
+				Spec: cmapi.CertificateSpec{
+					SecretName: "testcert-tls",
+					CommonName: s.newDomain(),
+					IssuerRef:  issuerRef,
+					Duration: &metav1.Duration{
+						Duration: time.Hour * 896,
+					},
+				},
+			}
+			By("Creating a Certificate")
+			err := f.CRClient.Create(ctx, testCertificate)
+			Expect(err).NotTo(HaveOccurred())
+
+			By("Waiting for the Certificate to be issued...")
+			err = f.Helper().WaitCertificateIssuedValid(f.Namespace.Name, "testcert", time.Minute*5)
+			Expect(err).NotTo(HaveOccurred())
+
+			f.CertificateDurationValid(testCertificate, time.Hour*896, 30*time.Second)
```

Comment:
What is `time.Hour * 896`?

Reply:
Also may be worth adding a comment explaining why we pass `time.Second * 30` here